ClimsoftVer4/ClimsoftVer4/frmNewHourlyWind.vb

```diff
@@ -162,7 +162,7 @@
         'COULD BE REMOVED IF ITS NOT NECESSARY
         'Check if header information is complete. If the header information is complete and there is at least on obs value then,
         'carry out the next actions, otherwise bring up message showing that there is insufficient data
-        If (Not ucrHourlyWind.IsDirectionValuesEmpty) And Strings.Len(ucrStationSelector.GetValue) > 0 And Strings.Len(ucrYearSelector.GetValue) > 0 And Strings.Len(ucrMonth.GetValue) And Strings.Len(ucrDay.GetValue) > 0 Then
+        If (Not ucrHourlyWind.IsDirectionValuesEmpty) AndAlso Strings.Len(ucrStationSelector.GetValue) > 0 AndAlso Strings.Len(ucrYearSelector.GetValue) > 0 AndAlso Strings.Len(ucrMonth.GetValue) AndAlso Strings.Len(ucrDay.GetValue) > 0 Then
```

Comment:
I don't think this is a sensible check to have. Can you remove it but leave the code in as comments so that if this is asked about it can easily be added back in. We should do this on all forms.

Reply:
The (Not ucrHourlyWind.IsDirectionValuesEmpty) or the entire if statement?

Reply:
The whole check. It doesn't seem sensible to prevent the user from clearing if data isn't entered, we can later check if there was reason for this and add back if needed.

Reply:
Done